database/cnl.sql

```diff
@@ -1884,5 +1884,35 @@ CREATE TABLE IF NOT EXISTS labor (
     FOREIGN KEY (work_order_id) REFERENCES work_orders(work_order_id)
 );
 
-
+CREATE TABLE `workflow_stages` (
+  `stage_id` char(36) NOT NULL,
+  `workflow_id` char(36) NOT NULL,
+  `stage_name` varchar(255) NOT NULL,
+  `stage_order` int NOT NULL,
+  `created_at` timestamp NULL DEFAULT CURRENT_TIMESTAMP,
+  `updated_at` timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`stage_id`),
+  KEY `workflow_id` (`workflow_id`),
+  CONSTRAINT `workflow_stages_ibfk_1` FOREIGN KEY (`workflow_id`) REFERENCES `workflows` (`workflow_id`)
+);
+ 
+CREATE TABLE `workflows` (
+  `workflow_id` char(36) NOT NULL,
+  `name` varchar(255) NOT NULL,
+  `created_at` timestamp NULL DEFAULT CURRENT_TIMESTAMP,
+  `updated_at` timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`workflow_id`)
+);
```

Comment:
Why are  ` ` added here?

Reply:
Actually, Thulasi, I forgot to add those two tables to the database file, so I took their structures from production and added them here.